src/translations/en/subscribe/confirm.mdx

```diff
@@ -0,0 +1,7 @@
+# Almost done!
+
+**You should receive an email with a confirmation link.**
```

Comment:
Isn't this a bit small compared to the main title? 🤔 

```suggestion
## You should receive an email with a confirmation link.
```

<img width="796" alt="Screen Shot 2022-06-12 at 22 38 29" src="https://user-images.githubusercontent.com/222463/173252781-8301f20e-ae11-48ca-ad96-f518419814f4.png">

Reply:
Yes, i corrected.